packages/server/test/util/factories.ts

```diff
@@ -43,7 +43,7 @@ export const CourseFactory = new Factory(CourseModel)
   .attr('icalURL', 'http://hi.com')
   .attr('enabled', true)
   .assocOne('semester', SemesterFactory)
-  .assocMany('officeHours', OfficeHourFactory);
+  .assocMany('officeHours', OfficeHourFactory, 0);
```

Comment:
Why is the 0 needed here? 

From my understanding this means no `officeHours` are being associated to a course. If that's the case could we remove the line all together?

Reply:
it needs to make an empty array i believe